ironfish-cli/Dockerfile

```diff
@@ -1,13 +1,17 @@
 FROM node:16.15.1-bullseye as build
 ENV PATH="/root/.cargo/bin:${PATH}"
 
-COPY ./ ./
-
 RUN \
     apt-get update && \
-    apt-get install jq rsync gcc cmake python -y && \
-    curl https://sh.rustup.rs -sSf | sh -s -- -y && \
-    ./ironfish-cli/scripts/build.sh
+    apt-get install jq rsync gcc cmake python -y
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+
+WORKDIR ./app
```

Comment:
This is just a minor tweak, made it a little cleaner when debugging, so I don't have to go "what is home and where did it come from" since it was building at / previously